server/views/components/formFieldList.njk

```diff
@@ -1,11 +1,9 @@
 {% from "./formField.njk" import formField %}
 
 {% macro formFieldList(fields) %}
-
-    <dl class="govuk-summary-list">
-        {% for field in fields %}
+    {% for field in fields %}
+        {% if field.value %}
```

Comment:
Need to remove the if condition. As values = 0 or false will not be rendered due to the falsy check. It has been agreed with the BAs, that we will render zeros. This will be handled in EMP-417.